src/test/java/org/jenkinsci/remoting/protocol/ProtocolStackImplTest.java

```diff
@@ -141,11 +141,8 @@ public void pipeCloseSource() throws Exception {
         pipe.source().close();
         assertThat(pipe.source().isOpen(), is(false));
         assertThat(pipe.sink().isOpen(), is(true));
-        try {
-            pipe.sink().write(ByteBuffer.allocate(1));
-        } catch (IOException e) {
-            assertThat(e.getMessage(), containsString("Broken pipe"));
-        }
+        final IOException e = assertThrows(IOException.class, () -> pipe.sink().write(ByteBuffer.allocate(1)));
```

Comment:
This change breaks the Windows build somehow; Linux works here though.

Reply:
> This change breaks

Therefore reverted in commit c336c910713a1b981cb0439beacbe5892dd5109a.

Reply:
Thanks!